### DISCUSSION TITLE
[Feature request] Group tasks

### DISCUSSION THREAD

**Discussion by christaikobo** [donetick/donetick#26] [Ideas]
It would be very beneficial to be able to group tasks.

Thank you!

**Comment by meauxt** [comment]
Hi @terrytw !
version 0.0.18 now has a better Label and Label manager. i been utilizing that for grouping things together

**Comment by christaikobo** [comment]
Hi thanks for the tips.

Labels and groups are not really the same thing. Label can be used as a workaround for sure.

If you don't intend to implement groups, you can close this issue.

Thanks again!

**Comment by meauxt** [comment]
@terrytw can you tell me what different behaviors to make sure I understand the different?
